src/main/resources/journals/journalList.csv

```diff
@@ -11801,12 +11801,12 @@ Physica Medica;Physica Med.;;
 Physica Scripta;Phys. Scr.;;
 Physica Scripta, T;Phys. Scr., T;;
 Physica Scripta, T.;Phys. Scr. T.;;
-Physica Status Solidi A: Applications and Materials Science;Phys. Status Solidi A;;
 Physica Status Solidi AApplications and Material Science;Phys. Status Solidi A;;
-Physica Status Solidi B: Basic Solid State Physics;Phys. Status Solidi B;;
+Physica Status Solidi A: Applications and Materials Science;Phys. Status Solidi A;;
```

Comment:
Shouldn't this line be deleted and in the line before `AAplications` replaced by `Aplications`? Similar question for the other two changes.

Reply:
The version is the colon is the right one, so I'll delete the other ones.